NEWS.md

```diff
@@ -1,3 +1,36 @@
+# testthat (development)
```

Comment:
```suggestion
# testthat (development version)
```

Since usethis is pretty strict about the format of this header when it is being auto updated from dev to release
https://github.com/r-lib/usethis/blob/486ed15e93c0d2421a623249441aa0685a124733/R/news.R#L33-L37

Reply:
This header is only indicative. We'll bump the version the usual way.